### PR TITLE
fix logs not loading

### PR DESCRIPTION
**Pull request by nanoNeel**
https://nanopay.atlassian.net/browse/CPF-3928
credits to @lchanmann for finding and fixing the problem.

### CODE REVIEW

File: src/foam/dao/WriteOnlyFileJournal.js [hunk 1]
```diff
@@ -32,8 +35,18 @@ foam.CLASS({
         final Object id = obj.getProperty("id");
 
         getLine().enqueue(new foam.util.concurrent.AbstractAssembly() {
+          FObject old;
           String record_ = null;
 
+          public Object[] requestLocks() {
+            return new Object[] { id };
```
Comment (kgrgreer) [inline src/foam/dao/WriteOnlyFileJournal.js]: Why is this needed?

Comment (kgrgreer) [reply src/foam/dao/WriteOnlyFileJournal.js]: requestLocks support isn't implemented yet, and if it were, this wouldn't work because you aren't guaranteed to get the exact same object every time for 'id', even if they do have the same value.

File: src/foam/dao/WriteOnlyFileJournal.js [hunk 2]
```diff
@@ -32,8 +35,18 @@ foam.CLASS({
...
+          }
+
+          public void executeUnderLock() {
+            old = dao.find_(x, id);
```
Comment (jlhughes) [inline src/foam/dao/WriteOnlyFileJournal.js]: 'old' is never used.
